security/docker/start_app.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+echo "Starting NodeAgent..."
+# Run node-agent
+/usr/local/bin/node_agent \
+--cert-chain /usr/local/bin/node_agent.crt \
+--key /usr/local/bin/node_agent.key \
+--root-cert /usr/local/bin/istio_ca.crt&
+
```

Comment:
Why is this in same container ? Why not have two?

Reply:
manages certificates using ca and it  should be running in the same container as the app.

Reply:
@rshriram This is testing the node agent can talk to CA and deliver the cert and key to the application, for VM use cases. We want to deploy the node agent + app in the Kubernetes pod to test this feature so a real VM is not needed in the e2e test.